cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunNativeImageTestTask.java

```diff
@@ -345,4 +402,229 @@ private String getClassPath(JBallerinaBackend jBallerinaBackend, Package current
         return classPath.toString();
     }
 
+    private boolean modifyJarForFunctionMock(TestSuite testSuite, Target target, String moduleName,
+                                             Map<String, String> functionMockModuleMapping) throws IOException {
+        String mainJarName = testSuite.getOrgName() + HYPHEN + moduleName + HYPHEN +
```

Comment:
We have `ProjectUtils.getJarFileName` which will return the name of the thin jar

Reply:
It (`getJarFileName(Module module)`) gives only the part of the module name.(Eg. `util`). With that partial name, we need to construct the name again. Other function (`getJarFileName(Package pkg)`) gives only the jar belongs to default module.